files/grest/rpc/pool/pool_metadata.sql

```diff
@@ -1,12 +1,26 @@
-CREATE OR REPLACE FUNCTION grest.pool_metadata(_pool_bech32 text)
-RETURNS JSON STABLE LANGUAGE PLPGSQL AS $$
+CREATE OR REPLACE FUNCTION grest.pool_metadata (_pool_bech32 text)
+    RETURNS json STABLE
+    LANGUAGE PLPGSQL
+    AS $$
 BEGIN
-    RETURN ( SELECT json_build_object(
-        'meta_url', pmd.url,
-        'meta_hash', encode(pmd.hash::bytea, 'hex')
-    )
-    FROM public.pool_meta_data AS pmd
-    WHERE pmd.id = (SELECT id from public.pool_hash where view=_pool_bech32)
-    );
-END; $$;
+    RETURN (
+        SELECT
+            json_build_object('meta_url', pmd.url, 'meta_hash', encode(pmd.hash::bytea, 'hex'))
+        FROM
+            public.pool_metadata_ref AS pmd
```

Comment:
For consistency in naming convention, should we change pmd to pmr?

Reply:
I was thinking about it but since `pmd` -> pool metadata, I thought it's fine. But I agree, we should abbreviate based on actual table name